recipes/epics-base/bld.bat

```diff
@@ -0,0 +1,16 @@
+if %ARCH%==32 (
+    set EPICS_HOST_ARCH=win32-x86
+) else if %ARCH%==64 (
+    set EPICS_HOST_ARCH=windows-x64
+)
+
+REM SCRIPTS causes failure of GNU make
+set SCRIPTS=
+
+REM set up build
+copy %RECIPE_DIR%\pre-build.py %SRC_DIR%
+%PYTHON% pre-build.py
```

Comment:
I have made an error here. Because this recipe does not declare dependency on python at build time, the %PYTHON% points to non-existing python.exe.

By replacing it with `python pre-build.py`, it will use the python in PATH, which is the SYS_PYTHON.

Reply:
@xiaoqiangwang, can we just specify a `build` dependency on `python`? Is the python version important?

Reply:
I think I understand now, apologies for the noise.